EIPS/eip-2228.md

```diff
@@ -20,22 +20,30 @@ This standard specifies the name for this network and provides reference example
 
 ## Motivation
 
-The Ethereum network with chainid 1 is referenced from authoritative sources using several names that are conflicting:
+The Ethereum network with chainid 1 is referenced using several conflicting names across EIPs (as published at the time EIP 2228 was written), client implementations, and information published on the internet at large:
 
-* Vitalik has called it "main net" at http://eips.ethereum.org/EIPS/eip-155 http://eips.ethereum.org/EIPS/eip-170
-* Vitalik has called it "mainnet" at http://eips.ethereum.org/EIPS/eip-86 
-* Vitalik has called it "Main net" at http://eips.ethereum.org/EIPS/eip-2* http://eips.ethereum.org/EIPS/eip-150
+* "main net" at [EIP-155](./eip-155.md), [EIP-170](./eip-170.md)
+* "mainnet" at [EIP-86 [DRAFT]](./eip-86.md), [client documentation](https://geth.ethereum.org/support/FAQ)
+* "Main net" at [EIP-2](./eip-2.md), [EIP-150](./eip-150.md)
    * \* In EIP-2, it is unclear if the first letter is capital because it is a proper noun or if it is the the first word in a text column presented in sentence case
-* Vitalik has called it "Mainnet" at http://eips.ethereum.org/EIPS/eip-7
-* Geth documentation calls its "mainnet" at https://geth.ethereum.org/support/FAQ
-* A final EIP here has referred to it as "Mainnet" at http://eips.ethereum.org/EIPS/eip-779
+* "Mainnet" at [EIP-7](./eip-7.md), [EIP-779](./eip-779.md)
+
+In several examples above, even papers written by the same author use (as published at the time EIP 2228 was written) inconsistent names to refer to the Ethereum network with chainid 1.
 
 ## Specification
 
 The network name shall be Ethereum Mainnet, or just Mainnet if the context is known to be discussing Ethereum networks. This IS a proper noun. Several examples are given below which differentiate between usage of the name of the network versus a descriptive reference to the network.
 
 Any name or word styling (i.e. capitalization of the letters) of the network which is inconsistent with the test cases cited below shall NOT be used.
 
+### Trademark note
+
+Note: Ethereum [is a registered trademark](https://tsdr.uspto.gov/#caseNumber=86634529&caseType=SERIAL_NO&searchType=statusSearch) of Ethereum Foundation and [the Foundation asserts](https://ethereum.org/en/terms-of-use/) "you must not use [this mark] without the prior written permission of the Foundation".
+
+Your application MAY reference "Ethereum&reg; Mainnet" and include a note "Ethereum is a registered trademark of Ethereum Foundation" elsewhere.
+
+Trademark status, fair use of trademarks, and any written permissions you have with the Ethereum Foundation are outside the scope of this EIP.
+
```

Comment:
The general EIP policy is to not include any external links.  Also, I feel like an EIP is not the appropriate place to mention trademarks or other legal statuses.  If people want to read the ToU they can read the ToU, no need to restate it here.  Your last line of this sums up my position pretty cleanly, but feels unnecessary to include in the EIP.
```suggestion
```